package.json

```diff
@@ -47,7 +47,7 @@
 		"@babel/preset-typescript": "^7.23.2",
 		"@babel/runtime": "^7.23.2",
 		"@changesets/cli": "^2.26.2",
-		"@guardian/eslint-config": "^4.0.0",
+		"@guardian/eslint-config": "^5.0.0",
 		"@guardian/eslint-config-typescript": "^6.0.0",
```

Comment:
As above, we only need one of these, and I’d recommend using `@guardian/eslint-config-typescript` v7